contracts/src/utils/protocol.ts

```diff
@@ -21,6 +21,8 @@ export const getProtocolManagerAddress = async(hre: HardhatRuntimeEnvironment):
 
     // For the tests we deploy a mock for the manager
     if (chainId === "31337") return deployMock(hre, "ManagerMock")
+
+    if (chainId === "5") return "0xb4Dc1B282706aB473cdD1b9899c57baD2BD3e2f3"
```

Comment:
Actually, this is the test manager contract: `0xAbd9769A78Ee63632A4fb603D85F63b8D3596DF9`

https://goerli.etherscan.io/address/0xAbd9769A78Ee63632A4fb603D85F63b8D3596DF9#code

https://github.com/5afe/safe-core-protocol/blob/main/deployments.ts#L1311